src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -370,7 +371,7 @@ paths:
         uri: '${base_uri}/eros/{eroId}/temporary-certificates'
         requestParameters:
           integration.request.path.eroId: method.request.path.eroId
-          integration.request.path.applicationId: method.request.path.applicationId
+          integration.request.querystring.applicationId: method.request.querystring.applicationId
```

Comment:
Is this needed?  (When Nathan added other query parameters I checked VCA spec and it didn't look like we specifically mapped query parameters)

Reply:
I am unsure 🤷🏼‍♂️  as applicationId is definitely not a `path` variable, so just corrected it to be a `querystring` param.

Reply:
I'm not sure either, but I'd prefer to have consistency with the other relevant endpoints as well.

Reply:
100% that it needs removing as a path variable (I must have forgotten that when I did it 😬 )
But I'm not sure about adding it as a querystring - I'm not sure it's needed ?
(When I made this change here in print-api to make this a query string param, I took my steer from the Search Application's endpoint in VCA ..... that endpoint has loads of query string params, and we dont define them in the APIG config)

Reply:
Thank you all . I have removed these and `applicationId queryString` won't be mapped in APIG integration as we all agreed/discussed. PR updated 👍🏼